### ISSUE TITLE
recognizer training  getbatch generator working endless 

### ISSUE THREAD

**Issue by rawat123** [faustomorales/keras-ocr#73]
hi,

i am trying to train recognizer width custom dataset  from this link https://keras-ocr.readthedocs.io/en/latest/examples/end_to_end_training.html using google colab
 while detector training worked fine but below code is running continuous and uses all RAM with google colab 
`max_length = 10
recognition_image_generators = [
    keras_ocr.data_generation.convert_image_generator_to_recognizer_input(
        image_generator=image_generator,
        max_string_length=min(recognizer.training_model.input_shape[1][1], max_length),
        target_width=recognizer.model.input_shape[2],
        target_height=recognizer.model.input_shape[1],
        margin=1
    ) for image_generator in training_image_generator 
]
`

below code is used to create image generator 
`import imgaug
train, validation = sklearn.model_selection.train_test_split(
    dataset_recognizer , train_size=0.8, random_state=42
)
augmenter = imgaug.augmenters.Sequential([
    imgaug.augmenters.Affine(
      scale=(1.0, 1.2),
      rotate=(-5, 5)
    ),
    imgaug.augmenters.GaussianBlur(sigma=(0, 0.5)),
    imgaug.augmenters.Multiply((0.8, 1.2), per_channel=0.2)
])
generator_kwargs = {'width': 200, 'height': 120}
training_image_generator = keras_ocr.datasets.get_detector_image_generator(
    labels=train,
    augmenter=augmenter,
    **generator_kwargs
)
validation_image_generator = keras_ocr.datasets.get_detector_image_generator(
    labels=validation,
    **generator_kwargs
)`

**Comment by faustomorales** [comment]
Closing due to inactivity and lack of detail (please see [contribution guidelines](https://github.com/faustomorales/keras-ocr#contributing)).
